src/index.js

```diff
@@ -94,14 +91,7 @@ const loadCargoWeb = async function (self, {release, name, target, regExp}, srcD
   const cmd = cargoCommand(target, release, ['web']);
   const result = await execPermissive(cmd, srcDir);
 
-  const {wasmFile, jsFile} = await handleCargo(self, result);
-
-  if (!wasmFile) {
```

Comment:
This can still happen even when Cargo doesn't throw an error; for example when the library is not a `"cdylib"` so please keep these lines.